go.mod

```diff
@@ -13,6 +13,7 @@ replace (
 )
 
 require (
+	github.com/agiledragon/gomonkey v2.0.2+incompatible
```

Comment:
why we need another version of gomonkey?

Reply:
`indirect` is generated automatically when running `go mod tidy`.
The `gomonkey` package in this project is a direct import, right?